modules/cnf-configuring-node-groups-for-the-numaresourcesoperator.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: PROCEDURE
+
+[id="cnf-configuring-node-groups-for-the-numaresourcesoperator_{context}"]
+= Configuring polling operations for NUMA resource updates
+
+You can configure how the NUMA Resources Operator polls a `nodeGroup` for available NUMA resources by configuring the `spec.nodeGroups` specification in the `NUMAResourcesOperator` custom resource (CR). This provides advanced control of NUMA Resources Operator polling operations. Configure these specifications to improve scheduling behaviour and troubleshoot suboptimal scheduling decisions. The configuration options are the following:
```

Comment:
this looks good _except_ the fact that:
1. the NUMA Resources Operator does watch the `spec.nodeGroups` in the `NUMAResorucesOperator` CR but
2. the NUMA Resources Operator USES this data to set the configuration of the daemonset pertaining to that nodeGroup
3. the daemons governed by the daemonset will in turn poll the kubelets on their nodes

So "NUMAResources polls a nodeGroup" is wrong.